### DISCUSSION TITLE
How to access ftp server with golang

### DISCUSSION THREAD

**Discussion by snorresovold** [pockethost/pockethost#197] [Q&A] [answered]
I'm using golang to try to acces the ftp server and make a script that automatically updates the schema. 

I'm using this code to try and access the server but I keep getting errors like "the specified class was not found" and "missing port in adress"

```
  package main
  
  import (
	  "log"
	  "github.com/jlaffaye/ftp"
	  "time"
  )
  
  func main() {
	  c, err := ftp.Dial("ftp://snorresovold%40gmail.com@pockethost.io", ftp.DialWithTimeout(5*time.Second))
	  if err != nil {
		  log.Fatal(err)
	  }
	  err = c.Login("anonymous", "anonymous")
	  if err != nil {
		  log.Fatal(err)
	  }
  
	  // Do something with the FTP conn
  
	  if err := c.Quit(); err != nil {
		  log.Fatal(err)
	  }
  }
```

What could I do to fix this? I think the error comes from a faulty Dial url.

**Comment by benallfree** [comment]
@snorresovold I am not super familiar with golang, so I can’t comment on that part. 

Are you able to access FTP and perform the desired operations via other tools? You might try that first just to verify that there isn’t something wrong with the FTP service itself (it’s a new feature).

**Reply by snorresovold** [reply, marked as answer]
I found a way to do what i wanted, currently working on a script that migrates the schema from 1 dev pocketbase to another prod pocketbase.
